security-analysis/security-analysis-api/src/main/java/com/powsybl/security/distributed/SecurityAnalysisExecutionHandler.java

```diff
@@ -100,9 +101,18 @@ public SecurityAnalysisExecutionHandler(ResultReader<R> reader,
      */
     @Override
     public R after(Path workingDir, ExecutionReport report) throws IOException {
-        super.after(workingDir, report);
-        LOGGER.debug("End of command execution in {}. ", workingDir);
-        return reader.read(workingDir);
+        try {
```

Comment:
I wonder : is it useful to not have the "super.after" and the "read" in the same try/catch ?
I would think that if the "super.after" throws, it's not useful to try to read the result (but of course it's still useful to create the ComputationException with the logs).
What do you think?

Reply:
Thank you for the comments :)
But I am still wondering :
in case of execution error of 1 command, do we want to have a "NoSuchFileException", or do we want to have a "the command *** has failed with exit code ***" ?
I would think the root cause is more the command failure, not the absence of result file which is the consequence, that's why checking the status before trying to read the results makes a little more sense to me.